src/lxml/html/__init__.py

```diff
@@ -287,15 +287,19 @@ def body(self):
         Return the <body> element.  Can be called from a child element
         to get the document's head.
         """
-        return self.xpath('//body|//x:body', namespaces={'x':XHTML_NAMESPACE})[0]
+        result = self.xpath('//body|//x:body', namespaces={'x':XHTML_NAMESPACE})
+        if len(result) > 1:
+            return result[0]
 
     @property
     def head(self):
         """
         Returns the <head> element.  Can be called from a child
         element to get the document's head.
         """
-        return self.xpath('//head|//x:head', namespaces={'x':XHTML_NAMESPACE})[0]
+        result = self.xpath('//head|//x:head', namespaces={'x':XHTML_NAMESPACE})
+        if len(result) > 1:
+            return result[0]
```

Comment:
Otherwise … do what? Return None? Would be good to say so if that's what you want.

Reply:
I've some difficulty to run lxml tests, it's a bit new for me, and that pull request is a kind of experience.  It should return None indeed, I can correct that.

Reply:
None

Reply:
11256de79711935479e9588aa26bc418cb8bb5ba

Reply:
> Otherwise … do what? Return None? Would be good to say so if that's what you want.

Reply:
> I've some difficulty to run lxml tests, it's a bit new for me, and that pull request is a kind of experience.  It should return None indeed, I can correct that.

Reply:
> 11256de79711935479e9588aa26bc418cb8bb5ba

https://github.com/lxml/lxml/pull/272#pullrequestreview-151623501

Reply:
> 11256de79711935479e9588aa26bc418cb8bb5ba

Reply:
> 11256de79711935479e9588aa26bc418cb8bb5ba

https://github.com/lxml/lxml

Reply:
> 11256de79711935479e9588aa26bc418cb8bb5ba